src/scss/components/button/mixins/_hxButton.scss

```diff
@@ -178,6 +178,7 @@
   background-color: transparent;
   border: none;
   color: $blue-500;
+  padding: 0;
```

Comment:
@lalithkarikelli, please check the tertiary top and bottom padding values.

### See `Tertiary Button Set`:

<img width="576" alt="Screen Shot 2021-04-18 at 11 37 03 PM" src="https://user-images.githubusercontent.com/10120600/115182579-d0e75d80-a09f-11eb-9b47-5432014e7ce6.png">

Reply:
Changes have been done, please review.